### DISCUSSION TITLE
系统托盘的图标是白色很难辨认

### DISCUSSION THREAD

**Discussion by shopkeeperV** [ClassicOldSong/Apollo#171] [Ideas]
![2024-12-26_19-57-12_lDJUs](https://github.com/user-attachments/assets/65db44c8-f0e6-4130-a0d5-4823dde1a97d)
你好如图第二个是apollo的白色图标，就像空了一块，很多人都是浅色的背景，因为windows默认就是浅色不透明，建议修改颜色或添加一个更改颜色的功能。

**Comment by ClassicOldSong** [comment]
默认已经添加阴影了，至少不是完全看不见（x

改图标比较麻烦，如果有更好的图标设计可以分享一下，如果合适的话可以考虑一起改了
